emf-builtins/build.gradle

```diff
@@ -0,0 +1,154 @@
+import org.apache.tools.ant.taskdefs.condition.Os
```

Comment:
For the other projects we are using .gradle.kts files. Would that be feasible to move it to a .gradle.kts? If it takes any significant amount of work, can you just open an issue for a follow up task for doing that?

Reply:
Atanas said the Eclipse Gradle plugin doesn't understand `.kts`. I didn't double-check, but assume he's correct.
And we definitely want to have the same build system.

Reply:
Right, then maybe we should add a note in the README about this